### PR TITLE
Enable audit streaming to Tools Audit Server

### PR DESCRIPTION
**Pull request by clloyd**
This adds support for @piuccio 's auditing setup. When enabled (All env's apart from DEV) when modifying a Tag or Section, a kinesis message is sent to the stream in the Fronts-CMS account.

cc @steppenwells

### CODE REVIEW

File: app/model/SectionAudit.scala [hunk 1]
```diff
@@ -20,6 +21,16 @@ case class SectionAudit(
                      sectionSummary: SectionSummary
                    ) {
   def toItem = Item.fromJSON(Json.toJson(this).toString())
+
+  def asAuditingThrift = Notification(
+    app = App.TagManager,
+    operation = operation,
+    userEmail = s"${user.replace(" ", ".").toLowerCase}@guardian.co.uk",
```
Comment (piuccio) [inline app/model/SectionAudit.scala]: is `user.email` not available? Some people have funny email, it's best to use whatever you use to authenticate against panda

File: app/model/SectionAudit.scala [hunk 2]
```diff
@@ -20,6 +21,16 @@ case class SectionAudit(
...
+    date = date.toString,
+    resourceId = Some(sectionId.toString),
+    message = Some(s"${description}. Section: ${sectionSummary.toString}"),
+    shortMessage = Some(s"${description} Section ID: ${sectionId.toString}")
```
Comment (piuccio) [inline app/model/SectionAudit.scala]: it's up to you to decide what to do with `message` and `shortMessage`, but maybe you only need `Some(s"${description}")`, the `sectionId` is already inside `resourceId`

**Comment by piuccio** [comment]
looks good

File: app/model/SectionAudit.scala (hunk 1 above)
Comment (clloyd) [inline app/model/SectionAudit.scala]: Unfortunately not at the moment (I've added a flaw friday item for me to switch to using that rather than name to identify users with our job system)

Comment (clloyd) [inline app/model/SectionAudit.scala]: Going to make this change as a separate PR now
